main.py

```diff
@@ -1,10 +1,25 @@
-from flask import Flask, render_template, request
+from flask import Flask, jsonify, render_template, request
+from flask_sqlalchemy import SQLAlchemy
 import json
 from flask_cors import CORS, cross_origin
 from hashlib import md5
 
-
+    
 app = Flask(__name__)
+
+#initialize configuration
+app.config["SQLALCHEMY_DATABASE_URI"] = "sqlite:///fallamoi.db"
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
```

Comment:
Add these in a configutation file, and import those from there !
it should not be as plain text like that in the codebase !

See this for example : https://gist.github.com/Sanix-Darker/bdaae2dfcb3ab74b356b4c3c799d7844